modules/ROOT/pages/cookies.adoc

```diff
@@ -0,0 +1,64 @@
+
+In general, websites have required cookies necessary for the site to work correctly, functional cookies necessary for certain site functionality, functional cookies that are not necessary for any site functionality, and advertising cookies.
+
+Exchange requires some cookies to work correctly, including cookies for session details and authentication information.
+
+Exchange normally uses some functional cookies, including cookies for site analytics. Exchange does not require any functional cookies for site functionality.
+
+Content in Exchange documentation often uses advertising cookies. If a video embedded in an Exchange portal is hosted on a third-party site such as YouTube that uses advertising cookies, then the video is not visible to a website visitor who disables advertising cookies, and the visitor sees a message explaining this.
+
+== Cookie Consent Management
+
+Exchange provides public portal website visitors with cookie consent management and full documentation of all cookies, in compliance with marketing and privacy regulations.
+
+Some countries require websites not to use any non-essential cookie until after the website visitor has granted explicit consent.
+
+When a user in a country with this requirement visits an Exchange public portal for the first time, Exchange notifies the user about the cookies stored in the browser, and the user accepts the default settings or opens the cookie consent manager to configure their cookie settings. Exchange finds the user's country by IP address.
+
+The Exchange cookie preferences button enables users in all countries to open the cookie consent manager and review or modify their settings at any time.
+
+The cookie consent manager shows settings for required cookies, functional cookies, and advertising cookies, and general information about cookies.
+
+Website visitors use the cookie consent manager to view the cookies the site uses, view descriptions of all individual cookies and understand what these cookies are doing, and enable or disable functional and advertising cookies.
+
+== Vanity Domains
+
+If you enable a vanity domain for your Exchange public portal, enable cookie consent management with the xref:portal-vanity-domain.adoc[public portal vanity domain instructions].
+
+== Cookies Used by Exchange
+
+[cols="1,1,1,5",options=header,width="100%",frame="topbot"]
+|===
+|Cookie Name |Duration |Cookie type |Description
+|__cfduid |Session |Required |Helps Cloudflare detect malicious visitors to our customers' websites and minimizes blocking legitimate users.
+|+__tld__+ |Session |Functional |
+|_csrf |Session |Required |Helps prevent Cross-Site Request Forgery attacks for site security.
+|_ga |730 days |Functional |Calculates visitor, session, and campaign data for site analytics.
+|_gat |Session |Functional |Throttles the request rate to limit the collection of data on high traffic sites.
+|_gid |1 day |Functional |
```

Comment:
`_gid` Description: "Registers a unique ID that is used to generate statistical data on how the visitor uses the website."